numpy/core/src/_simd/_simd.c

```diff
@@ -0,0 +1,43 @@
+#include "_simd.h"
+
+PyMODINIT_FUNC PyInit__simd(void)
+{
+    static struct PyModuleDef defs = {
+        .m_base = PyModuleDef_HEAD_INIT,
+        .m_name = "_simd",
+        .m_size = -1
+    };
+    if (npy_cpu_init() < 0) {
+        return NULL;
+    }
+    PyObject *m = PyModule_Create(&defs);
+    if (m == NULL) {
+        goto err;
+    }
+    PyObject *targets = PyDict_New();
+    if (targets == NULL) {
+        goto err;
+    }
+    NPY_CPU_DISPATCH_CALL_ALL(simd_attach_module, (targets))
+    // Add keys for non-supported optimizations with None value
+    #define CHECK_NON_SUPPORTED(TESTED_FEATURES, TARGET_NAME, ...) \
+        if (!TESTED_FEATURES) { \
+            Py_INCREF(Py_None); \
+            if (PyDict_SetItemString(targets, NPY_TOSTRING(TARGET_NAME), Py_None) < 0) { \
+                Py_DECREF(Py_None); Py_DECREF(targets); goto err; \
+            } \
+        }
+    NPY__CPU_DISPATCH_CALL(NPY_CPU_HAVE, CHECK_NON_SUPPORTED, DUMMY_4MSVC)
+
+    if (PyErr_Occurred() || PyModule_AddObject(m, "targets", targets) < 0) {
+        Py_DECREF(targets);
+        goto err;
+    }
+    return m;
+err:
+    if (!PyErr_Occurred()) {
+        PyErr_SetString(PyExc_RuntimeError, "cannot load _simd module.");
```

Comment:
How is this line reachable?

Reply:
mostly a null argument or unhandled error. hopefully, unexpected behavior not related to this patch :).